index.html

```diff
@@ -196,5 +196,39 @@ <h2>About me</h2>
     </div>
   </section>
 
-  
+ <!-- contact and form section -->
+
+ <section class="contact-section">
+    <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    <form class="contact-form" action="https://formspree.io/f/xwkjrkqw" method="post" id="form-element">
+      <ul>
+        <li>
+          <label for="full-name"></label>
+          <input id="full-name" type="text" name="name" placeholder="Full name" maxlength="30" required />
+        </li>
+        <li>
+          <label for="email"></label>
+          <input id="email" type="email" name="email" placeholder="Email address" required />
```

Comment:
- [ ] Please either remove the labels from your code because you don't have the `labels` heading for the inputs **Or** you can wrap your inputs with a label like the below example which gives your code more structure, looks clean and easy to understand.

```
<label>
   <input ..... />
</label>
```